app/controllers/api/v1/instances/activity_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class Api::V1::Instances::ActivityController < Api::BaseController
+  before_action :require_enabled_api!
+
+  respond_to :json
+
+  def show
+    render_cached_json('api:v1:instances:activity:show', expires_in: 1.day) { activity }
+  end
+
+  private
+
+  def activity
+    weeks = []
+
+    12.times do |i|
+      week    = i.weeks.ago.to_date
+      week_id = week.cweek
+
+      weeks << {
+        week: week.to_time.to_i.to_s,
```

Comment:
This should probably either be `week_id` or a timestamp for the start of the week, not just a day in that week.

Reply:
It is the timestamp of the start of the week